### PR TITLE
Feat/created UI

### PR DESCRIPTION
**Pull request by prabhat-wednesday**
### Ticket Link

---

### Related Links

---

### Description
Created UI and render the data on the screen
---

### Steps to Reproduce / Test

---

---

### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added]
- [ ] Relevant documentation is changed or added (and PR referenced)

### GIF's
Screenshot 2022-09-30 at 3 10 20 PM
Screenshot 2022-09-30 at 3 10 42 PM
Screenshot 2022-09-30 at 3 11 06 PM

---

### CODE REVIEW

File: app/app.tsx [hunk 1]
```diff
@@ -27,11 +27,11 @@ import '!file-loader?name=[name].[ext]!./images/favicon.ico';
 import 'file-loader?name=.htaccess!./.htaccess';
 /* eslint-enable import/no-unresolved, import/extensions */
 
-import configureStore from './configureStore';
-
 // Import i18n messages
 import { translationMessages } from './i18n';
 import { Router } from 'react-router-dom';
+// eslint-disable-next-line import/no-duplicates
```
Comment (christin-wednesday) [inline app/app.tsx]:
```suggestion
```

File: app/containers/ItuensContainer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        func.apply(context, args);
+      }, 500);
+    };
+  };
+
+  const handleOnChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const artistSearch = e.target.value;
+    if (!isEmpty(artistSearch)) {
+      setArtistName(artistSearch);
+      dispatch(getSearchTerm(e.target.value));
```
Comment (christin-wednesday) [inline app/containers/ItuensContainer/index.tsx]: let's stick to the dispatch prop that is mapped

File: app/containers/ItuensContainer/reducer.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export interface ApiResponseState {
+  loading: boolean;
+  dataToShow: [];
```
Comment (christin-wednesday) [inline app/containers/ItuensContainer/reducer.ts]: we should specify the type of dataToShow.

**Review by christin-wednesday — CHANGES_REQUESTED** [review]

File: app/containers/ItuensContainer/apiUtils.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { create } from 'apisauce';
+
+const api = create({ baseURL: 'https://itunes.apple.com' });
```
Comment (christin-is-buildding) [inline app/containers/ItuensContainer/apiUtils.ts]: apiUtils will not be container specific, this should be a separate file in the services folder

File: app/containers/ItuensContainer/index.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+import { getSearchTerm, deleteResponse } from './reducer';
+import { useDispatch, connect } from 'react-redux';
+import ituneCallSaga from './saga';
+import { injectSaga } from 'redux-injectors';
+import { AnyAction, compose } from '@reduxjs/toolkit';
+import { selectError, selectLoading, selectDataToShow } from './selector';
+import { isEmpty } from 'lodash-es';
+import LoadAbleCard from '@app/components/Card';
+
+const ItunesContainer = ({ dispatchArtistName, dataToShow }: any) => {
```
Comment (christin-is-buildding) [inline app/containers/ItuensContainer/index.tsx:13]: please add tests

**Review by christin-is-buildding — CHANGES_REQUESTED** [review]
please add screen shots

**Review by christin-is-buildding — APPROVED** [review]

**Pull request merged**